vital/algo/video_input.h

```diff
@@ -374,6 +374,26 @@ class VITAL_ALGO_EXPORT video_input
   virtual kwiver::vital::metadata_vector frame_metadata() = 0;
 
 
+  /**
+   * \brief Get frame rate from the video.
+   *
+   * If frame rate is not supported, return -1.
```

Comment:
For a `double`, consider "a negative value" instead of "-1" to avoid potentially encouraging folks to do an equality comparison. (Note: still *actually return* `-1`; the suggestion is only a documentation change.)